samples/cpp/create_mask.cpp

```diff
@@ -123,14 +123,16 @@ void mouseHandler(int event, int x, int y, int, void*)
 
 int main(int argc, char **argv)
 {
-
-    if(argc != 2)
+    cv::CommandLineParser parser(argc, argv, "{@input||}");
+    if (!parser.has("@input"))
```

Comment:
Currently, positional parameters existence check does not work this way. You should either add this functionality to CommandLineParser, either use other way (like in [example](http://docs.opencv.org/master/d0/d2e/classcv_1_1CommandLineParser.html#details)): 
- first call all `get` methods
- then check parser error state:
```.cpp
if (!parser.check())
{
    parser.printErrors();
    return 0;
}
```